<ui version="4.0" >
 <class>SongBookDialog</class>
 <widget class="QDialog" name="SongBookDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Book Song Maintenance</string>
  </property>
  <layout class="QVBoxLayout" name="DialogLayout" >
   <property name="spacing" >
    <number>8</number>
   </property>
   <property name="margin" >
    <number>8</number>
   </property>
   <item>
    <widget class="QTableWidget" name="BookSongListView" />
   </item>
   <item>
    <widget class="QGroupBox" name="SongBookGroup" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="title" >
      <string>Song Book</string>
     </property>
     <layout class="QVBoxLayout" name="SongBookLayout" >
      <property name="spacing" >
       <number>8</number>
      </property>
      <property name="margin" >
       <number>8</number>
      </property>
      <item>
       <widget class="QWidget" native="1" name="DetailsWidget" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QFormLayout" name="DetailsLayout" >
         <property name="horizontalSpacing" >
          <number>8</number>
         </property>
         <property name="verticalSpacing" >
          <number>8</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="NameLabel" >
           <property name="text" >
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="NameEdit" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="PublisherLabel" >
           <property name="text" >
            <string>Publisher:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="PublisherEdit" />
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="MessageLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" native="1" name="ButtonWidget" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="ButtonLayout" >
         <property name="spacing" >
          <number>8</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>61</width>
             <height>24</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ClearButton" >
           <property name="toolTip" >
            <string>Clear Selection</string>
           </property>
           <property name="text" >
            <string>New</string>
           </property>
           <property name="icon" >
            <iconset resource="../images/openlp-2.qrc" >
             <normaloff>:/services/service_new.png</normaloff>:/services/service_new.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddUpdateButton" >
           <property name="toolTip" >
            <string>Add Update Author</string>
           </property>
           <property name="text" >
            <string>Save</string>
           </property>
           <property name="icon" >
            <iconset resource="../images/openlp-2.qrc" >
             <normaloff>:/services/service_save.png</normaloff>:/services/service_save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="DeleteButton" >
           <property name="toolTip" >
            <string>Delete Author</string>
           </property>
           <property name="text" >
            <string>Delete</string>
           </property>
           <property name="icon" >
            <iconset resource="../images/openlp-2.qrc" >
             <normaloff>:/services/service_delete.png</normaloff>:/services/service_delete.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>SongBookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>SongBookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
